packages/app-account/src/interbit/github-kyc/index.js

```diff
@@ -40,10 +41,11 @@ const initialState = Immutable.from({
     profileUrl: 'https://api.github.com/user',
     callbackUrl: process.env.OAUTH_CALLBACK_URL
   },
+
   // User profiles by chain ID. State shape is:
   //
   // profiles: {
-  //   [consumerChainId]: {
+  //   [privateChainId]: {
```

Comment:
This would be better in a unit test

Reply:
This is contained in the saga test for an existing private chain.